engine/runtime/src/main/java/org/enso/interpreter/node/callable/function/BlockNode.java

```diff
@@ -30,6 +30,10 @@ public static BlockNode build(ExpressionNode[] expressions, ExpressionNode retur
     return new BlockNode(expressions, returnExpr);
   }
 
+  public static BlockNode buildSilent(ExpressionNode[] expressions, ExpressionNode returnExpr) {
```

Comment:
Why does this exist?

Reply:
This one is used from `AtomConstructor.buildConstructorFunction`. Those functions are usually quite small - not sure it makes sense to make them lazy.